### PR TITLE
WIP: My Area

### PR DESCRIPTION
**Pull request by skamoen**

### CODE REVIEW

File: app/elements/lancie-my-area/my-area-orders.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    paper-card {
+      width: 100%;
+    }
+    
+    paper-tooltip {
+      --paper-tooltip-background: #1A2B43;
+      --paper-tooltip-text-color: #FEE474;
+    }
+    </style>
+    <paper-card heading="Orders & Tickets" elevation="1" animated-shadow="true">
```
Comment (TimvdLippe) [inline app/elements/lancie-my-area/my-area-orders.html]: &amp;

File: app/elements/lancie-my-area/my-area-teams.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+
+<link rel="import" href="../../bower_components/paper-card/paper-card.html">
+<!-- <link rel="import" href="../../bower_components/paper-item/paper-icon-item.html"> -->
```
Comment (TimvdLippe) [inline app/elements/lancie-my-area/my-area-teams.html]: Remove unused imports (or uncomment them)

**Comment by Icety** [comment]
Ready for pull now in my opinion

**Comment by skamoen** [comment]
It's... functional.

**Pull request merged**